modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -864,6 +864,170 @@ private Map<String, String> _addAction(
 			actionName, methodName, serviceBuilderObjectEntry, null, uriInfo);
 	}
 
+	private com.liferay.object.model.ObjectEntry _addNestedObjectEntries(
```

Comment:
I think this is something we should talk internally because when I talked about this with @sergiojimcos , we didn't fully know if what's currently happening is the expected behavior. I will open a thread in Slack.